moonscript/cmd/moonc.moon

```diff
@@ -70,6 +70,13 @@ compile_file_text = (text, opts={}) ->
   parse_time = if opts.benchmark
     assert gettime!
 
+  if opts.fname and string.sub(opts.fname,-8)==".litmoon"
```

Comment:
this same code appears to be duplicated in `moonscript/base.moon`, maybe it makes sense to put this in a module as a function somewhere?